bokeh/models/widgets/markups.py

```diff
@@ -94,6 +94,9 @@ class Div(Markup):
 
     This Bokeh model corresponds to an HTML ``<div>`` element.
 
+    Content can be interpreted as \
+    `TeX and LaTeX input <https://docs.mathjax.org/en/latest/basic/mathematics.html#tex-and-latex-input/>` \
+    setting enable_tex to ``True``
```

Comment:
more consistent with previous practice:
```suggestion
    Content can be interpreted as `TeX and LaTeX input`_ by  setting to True`
    
    .. _`TeX and LaTeX input`: https://docs.mathjax.org/en/latest/basic/mathematics.html#tex-and-latex-input
    
```